### PR TITLE
OracleDB `k8s` plugin support

### PR DESCRIPTION
**Pull request by algchoo**
### Description
This PR updates the OracleDB mixin to support the `grafana-k8s-plugin`

### Changes
* [updated logs expression to include more directories](https://github.com/grafana/jsonnet-libs/commit/0cd846b81c778be6c5914f2632b267097a8e4dea)
* [wip oracledb mixin updates](https://github.com/grafana/jsonnet-libs/commit/2426604c2fcbb81a5e3342b45074b08cb1479e42)
* [replaced static matcher with getMatcher and updated panels to take in matcher argument](https://github.com/grafana/jsonnet-libs/commit/d8ccf4b4f016fd674988d15e3a82e3602e2e7616) 
* [updated panels in addPanels to use getMatcher](https://github.com/grafana/jsonnet-libs/commit/3687883af725d49729907fdcec19028ed0c32ed8)

### CODE REVIEW

File: oracledb-mixin/dashboards/overview.libsonnet [hunk 1]
```diff
@@ -5,7 +5,7 @@ local dashboardUid = 'oracledb-overview';
 
 local prometheus = grafana.prometheus;
 local promDatasourceName = 'prometheus_datasource';
-local matcher = 'job=~"$job", instance=~"$instance"';
+local getMatcher(cfg) = '%(tensorflowSelector)s, instance=~"$instance"' % cfg;
```
Comment (StefanKurek) [inline oracledb-mixin/dashboards/overview.libsonnet:8]: tensorflowSelector to `oracledbSelector`. Also make sure that you run this in a local cluster with `enableMultiCluster` set to both true and false and see changes (new selector & panel queries have extra `cluster` filter)

File: oracledb-mixin/dashboards/overview.libsonnet [hunk 2]
```diff
@@ -1113,7 +1113,7 @@ local tablespaceSizePanel = {
               template.new(
                 'tablespace',
                 promDatasource,
-                query='label_values(oracledb_tablespace_bytes{' + matcher + '}, tablespace)',
+                query='label_values(oracledb_tablespace_bytes{%(oracledbSelector)s}, tablespace)' % $._config,
```
Comment (StefanKurek) [inline oracledb-mixin/dashboards/overview.libsonnet:1116]: any reason why this didn't stay as "matcher"?

**Review by StefanKurek — CHANGES_REQUESTED** [review]

File: oracledb-mixin/dashboards/overview.libsonnet (hunk 2 above)
Comment (algchoo) [reply oracledb-mixin/dashboards/overview.libsonnet:1116]: I replaced the static `matcher` variable with the `getMatcher` function. Not sure if function is the appropriate term. 

With `matcher` not existing anymore, this looks at the config and pulls selectors from there, to my knowledge. The portions that still contain `' + matcher + '` are going to be the panels that take that in as an argument.

In the `addPanels` section, the `getMatcher` function takes in the config and returns the appropriate selectors.
```
databaseStatusPanel(getMatcher($._config))
```

Comment (StefanKurek) [reply oracledb-mixin/dashboards/overview.libsonnet:1116]: @algchoo sure, I get that. But why did this query switch from using `matcher` to `oracledbSelector`?

Comment (algchoo) [reply oracledb-mixin/dashboards/overview.libsonnet:1116]: @StefanKurek The variable `matcher` no longer exists, so we have to look elsewhere for the selectors. I'm not sure I could use the `getMatcher` function and this is the [example](https://github.com/grafana/jsonnet-libs/pull/1158/files) I have been referring to. Is there a different approach I might take here?

Comment (algchoo) [reply oracledb-mixin/dashboards/overview.libsonnet:1116]: Also, just noticed that this replacement nerfs including `instance` as part of the selectors, added that back.

Comment (StefanKurek) [reply oracledb-mixin/dashboards/overview.libsonnet]: @algchoo ok sorry. Was missing that matcher wasn't being passed in here.

**Review by Dasomeone — APPROVED** [review]
LGTM, starting CI :)

**Pull request merged**